wordpress-mobile/WordPress-iOS
# Migrates domains service remote to WordPressKit

Fixes #7309 

This PR:
* Adds a new `RemoteDomain` model to the WPKit framework
* Migrates `DomainsServiceRemote.swift` to the WPKit framework
* Adds new unit tests (with stubs) for DomainsServiceRemote.swift to the WPKit framework (`DomainsServiceRemoteRESTTests`)

Ref #7301

**To test:**
1. Clean DerivedData.
2. Update CocoaPods repo with pod repo update.
3. Install the pods with pod install.
4. Run _all_ unit tests and make sure they are green.
5. Build & run the app — make sure everything works

Needs review: @astralbodies
